pybossa/jobs.py

```diff
@@ -1355,6 +1355,25 @@ def delete_file(fname, container):
     from pybossa.core import uploader
     return uploader.delete_file(fname, container)
 
+def load_usage_dashboard_data(min_date):
+
+    # total tasks, taskruns, projects over a specified amount of time.
+    timed_stats_funcs = [
+        site_stats.total_active_projects,
+        site_stats.total_tasks,
+        site_stats.total_taskruns,
+    ]
+
+    stats = OrderedDict()
+    for func in timed_stats_funcs:
+        key = func.__doc__.strip().title()
+        stats[key] = func(min_date)
+
+    # component usage
+    for name, tag in current_app.config.get("USAGE_DASHBOARD_COMPONENTS", {}).items():
+        stats['Projects Using ' + name] = site_stats.total_projects_using_component(tag, min_date)
```

Comment:
Could we flatten `stats['Projects Using ' + name]` into simply `stats[name]` and allow the js code to format the text display by iterating over the keys and rendering?

```js
for (const key in stats) {
  console.log(`Projects Using ${key}: stats[key]`);
}
```

Reply:
Resolved via https://github.com/bloomberg/pybossa/pull/824/commits/94b92585fbd7476dad9208cafb336f0211931534